src/setup/index.android.js

```diff
@@ -0,0 +1,8 @@
+import {Platform, UIManager} from 'react-native';
+
+export default function () {
+    // To use layout animations in Android: https://reactnative.dev/docs/layoutanimation
+    if (Platform.OS === 'android' && UIManager.setLayoutAnimationEnabledExperimental) {
+        UIManager.setLayoutAnimationEnabledExperimental(true);
+    }
+}
```

Comment:
I can drop my commit adding LayoutAnimation. It does look nicer on iOS though

Reply:
Yeah, I'd say there is plenty to focus on here already. But if we want to propose this as a follow up improvement it's likely people would get on board with the suggestion.